test/contrib/rails/controller_test.rb

```diff
@@ -21,12 +21,9 @@ class TracingControllerTest < ActionController::TestCase
     assert_equal(spans.length, 2)
 
     span = spans[1]
-    assert_equal(span.name, 'rails.request')
+    assert_equal(span.name, 'rails.action_controller')
     assert_equal(span.span_type, 'http')
     assert_equal(span.resource, 'TracingController#index')
```

Comment:
yes, all http-related tags are handled in the Rack middleware. I think this is the right approach, especially because a middleware can still change the `status_code` when the middleware stack is executed "backward" (in the sense of code after the `@app.call()`).